src/main/java/com/aws/greengrass/clientdevices/auth/connectivity/HostAddress.java

```diff
@@ -21,6 +29,14 @@ public static HostAddress of(ConnectivityInfo connectivityInfo) {
         return HostAddress.of(connectivityInfo.hostAddress());
     }
 
+    /**
+     * Checks if the address represented by this HostAddress is valid.
+     * @return true if valid
+     */
+    public boolean isValid() {
```

Comment:
why are only IPs and localhost valid? What about other DNS names?

Reply:
yeah that was a miss, removed the check